proxy/http/HttpSM.cc

```diff
@@ -191,6 +191,26 @@ HttpVCTable::remove_entry(HttpVCTableEntry *e)
     free_MIOBuffer(e->write_buffer);
     e->write_buffer = nullptr;
   }
+  if (e->read_vio != nullptr && e->read_vio->cont == sm) {
+    // Cleanup dangling i/o
+    if (e == sm->get_ua_entry() && sm->get_ua_txn()) {
+      e->read_vio = sm->get_ua_txn()->do_io_read(NULL, 0, NULL);
```

Comment:
No `NULL`.

Reply:
The `read_vio` will get cleaned out later, so this doesn't repeatedly shut down IO?